### PR TITLE
Cleanup sparse tests

### PR DESCRIPTION
**Pull request by fmassa**
## What does this PR do?

With #202 merged, we can now have a single entry-point for testing that sparse tensors behave as expected, with a unified set of tests to cover the required operators.

I've removed `test_sparse_csr.py` as it is now redundant compared to `test_sparse_tensors.py`.

I think a next step would be to clean `test_custom_ops.py` so that they test the bare minimum and doesn't depend on the legacy `SparseCS` tensor.

### CODE REVIEW

**Comment by fmassa** [comment]
Looks like doc failures are unrelated?

**Comment by blefaudeux** [comment]
> Looks like doc failures are unrelated?

Doc failure now fixed :) unrelated indeed

File: xformers/sparse/blocksparse_tensor.py [hunk 1]
```diff
@@ -114,8 +114,7 @@ def __init__(self, values, layout):
     def __repr__(self):
         return f"block_sparse_tensor(shape={self.shape}, values={self.__values})"
 
-    @property
-    def _blocksparse_values(self):
+    def values(self):
         return self.__values
```
Comment (fmassa) [inline xformers/sparse/blocksparse_tensor.py:118]: We now unify obtaining the values of the sparse tensor under the same API, which follows the API from PyTorch

**Comment by fmassa** [comment]
Merging, as this is a harmless PR wrt functionality.

**Pull request merged**